Directory.Build.targets

```diff
@@ -6,20 +6,11 @@
     <Exclude>[GraphQL.Samples.Schemas.Chat]*,[GraphQL.Samples.Server]*</Exclude>
   </PropertyGroup>
 
-  <PropertyGroup Condition="'$(OutputType)' == 'Exe'">
-    <IsPackable>false</IsPackable>
-  </PropertyGroup>
-  
```

Comment:
This likely has to be reverted after `<IsPackable>true<IsPackable>` is reverted

Reply:
https://github.com/graphql-dotnet/server/pull/910#discussion_r951075226

Reply:
This could be moved in to Samples.props now.

Reply:
I added IsPackable=false into samples.props.